rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -458,7 +458,7 @@ namespace rsx
 
 				// Get real args count
 				const u32 count = std::min<u32>({rsx->fifo_ctrl->get_remaining_args_count() + 1,
-					static_cast<u32>(((rsx->ctrl->put & ~3ull) - (rsx->fifo_ctrl->get_pos() - 4)) / 4), 32 - index});
+					static_cast<u32>(((rsx->ctrl->put & ~3ull) - (rsx->fifo_ctrl->get_pos())) / 4), 32 - index});
```

Comment:
Let's name these variables. There's also an implicit reliance on unsigned underflow here, a comment would be nice.